.travis.yml

```diff
@@ -68,5 +66,9 @@ jobs:
       name: "Vault"
     - script: npm run coverage:voting
       name: "Voting"
+
+    - stage: integration
+      script: npm run test:payroll:except:payday
+      name: "Payroll payday"
```

Comment:
This file is a bit confusing, the `name` key corresponds to the `script` right above haha
I think you wanted to move `npm run test:payroll:payday` to a separate stage

Reply:
Yes... good catch!